src/components/search.scss

```diff
@@ -1,5 +1,81 @@
-@use '../lib/theme';
+@use '../lib/theme.scss';
 
-.header {
+.searchWrapper {
+  margin: 1em 0em;
+}
+
+.searchWrapper__input {
+  width: 100%;
+  border: none;
+  border-radius: 35px;
+  outline: none;
+  color: theme.$primary-color;
+  box-shadow: 0px 0px 10px 5px rgba(0, 0, 0, 0.2);
+  padding: 0.5em 1em;
+  font-size: 1em;
+  font-weight: 100;
+  text-align: center;
+}
+
+.searchResultsWrapper {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  margin-left: -0.5%;
+  margin-right: -0.5%;
+}
+
+.searchResultsWrapper__searchResult {
+  border: none;
+  border-radius: 5px;
+  outline: none;
+  cursor: pointer;
+  padding: 0.5em 1em;
+  font-size: 1em;
+  font-weight: 100;
+  box-shadow: 0px 0px 5px 0px rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
+}
+
+.searchResultsWrapper__searchResult:hover {
+  color: theme.$alternate-color;
   background-color: theme.$secondary-bg-color;
 }
+
+.searchResultsWrapper__title {
+  font-weight: 100;
+  text-align: center;
+}
+
+.searchResultsWrapper__imageWrapper {
+  display: flex;
+  justify-content: center;
+}
+
+.searchResultsWrapper__image {
+  height: 200px;
+}
+
+/* Mobile */
+@media only screen and (max-width: 600px) {
+  .searchResultsWrapper__searchResult {
+    flex-basis: 100%;
+    margin: 0.5em 0;
+  }
+}
+
+/* Tablet */
+@media only screen and (min-width: 601px) and (max-width: 1024px) {
+  .searchResultsWrapper__searchResult {
+    flex-basis: calc(97% / 3);
```

Comment:
These calculations look weird. 
I would suggest to have a fixed margin between the items and set the flex-basis like this:

```
flex-basis: calc(100% / 3 - 10px);
margin: 5px;
```

Reply:
I tried to make everything use 100% width (margins + flex items combined). Now using fixed px margins -> https://github.com/hauptdigital/redesigned-pokedex/pull/6/commits/caa8e63cdcda1808ee113d7aed0c2193e198e401